app/code/Magento/Inventory/Model/PackageInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Model;
+
+interface PackageInterface
+{
+    /**
+     * @return \Magento\InventoryApi\Api\Data\SourceInterface
+     */
+    public function getSource();
+
+    /**
+     * @return \Magento\Quote\Model\Quote\Item[]
+     */
+    public function getItems();
+
+    /**
+     * @return \Magento\Quote\Model\Quote\Address
+     */
+    public function getAddress();
+
+    /**
+     * @param $address \Magento\Quote\Model\Quote\Address
+     * @return $this
+     */
+    public function setAddress($address);
+
+    public function getQty();
```

Comment:
Doesn't it make sense to just have different quote items in different packages? As it is already implemented even in the current version of the Shipping Assignments